elixir/apps/web/lib/web/live/policies/edit.ex

```diff
@@ -1,20 +1,28 @@
 defmodule Web.Policies.Edit do
   use Web, :live_view
 
+  alias Domain.Policies
+
+  def mount(%{"id" => id} = _params, _session, socket) do
+    {:ok, policy} = Policies.fetch_policy_by_id(id, socket.assigns.subject)
+
+    {:ok, assign(socket, policy: policy)}
```

Comment:
It would be better if we return 404 instead of crashing and returning 500 for non existing policies:
```suggestion
    with {:ok, policy} <- Policies.fetch_policy_by_id(id, socket.assigns.subject) do
      {:ok, assign(socket, policy: policy)}
    else
      _other -> raise Web.LiveErrors.NotFoundError
    end
```

Reply:
Make sense.  Appreciate the info!  This should be updated now.